doc/plugin_server_upstreamauthority_awssecret.md

```diff
@@ -12,6 +12,7 @@ The plugin accepts the following configuration options:
 | region            | AWS Region that the AWS Secrets Manager is running in |
 | cert_file_arn     | ARN of the "upstream" CA certificate                  |
 | key_file_arn      | ARN of the "upstream" CA key file                     |
+| bundle_file_arn   | ARN of additional roots to include in the trust bundle. If `cert_file_arn` contains a self-signed certificate this field can be left unset. Otherwise, `bundle_file_arn` must include one or more certificates necessary to establish a valid chain up to a self-signed certificate. |
```

Comment:
```suggestion
| bundle_file_arn   | ARN of roots to include in the trust bundle. If `cert_file_arn` contains a self-signed root CA certificate this field can be left unset. Otherwise, `bundle_file_arn` must include one or more root CA certificates |
```